docs/introduction.rst

```diff
@@ -82,27 +82,66 @@ approaches to this:
+
 Let's take a look at the pros and cons of these two approaches.
 
 White Box Methods
 ^^^^^^^^^^^^^^^^^
 
-*Explanation options are correlated to how the model operation can be
-inspected and interpreted. pros: closely tied to the individual model, thus can leverage
-specific knowledge about its architecture. Measures the actual
-computation which generates the output. Cons: specific to individual
-models / classes of models; harder to generalize across models.
-Requires modifying the model implementation to gain access; may
-require updating as the implementation evolves.*
+A **white box** model exposes some (or all) of the internal state of
+the model; the explanation draws a connection between this exposed
+state and the model output. Some models are intrinsically introspective to the point where they are not so much "white box" as transparent:
+
+* In `linear regression
+  <https://en.wikipedia.org/wiki/Linear_regression>`_ the influence of
+  the observations on the solution are an immediate consequence of the
+  method.
+
+* In a `decision tree <https://en.wikipedia.org/wiki/Decision_tree>`_,
+  the output is directly computed by making the comparisons and
+  branches encoded in the model.
+
+An example of a white box explanation method for CNNs is Grad-CAM (`paper <https://arxiv.org/abs/1610.02391>`_, `code <https://github.com/ramprs/grad-cam/>`_), which exposes some (but not all) of the CNN layers to the explanation generation algorithm.
+
+.. figure:: figures/intro-grad-cam-annotated.png
+
+   Figure 5: Grad-CAM architecture. Functional regions annotated to align with Figure 4: input in green, CNN architecture in yellow, output in orange. The explanation functionality is in blue; the APIs granting access to the model are in red. Figure from the `Grad-CAM code repo. <https://github.com/ramprs/grad-cam/>`_
+
+In Grad-CAM, first the model computes its output per the typical processing flow: input (green), through the CNN (yellow), to the output (orange.) The explanation is created via an additional processing step which uses the output and feature maps from within the CNN (red circles) to measure and visualize the activation of those regions associated with the output (blue).
+
+Two aspects typical of white box methods are demonstrated here:
+
+* **The explanation could not have been created from the output alone**. In order to operate, the explanation algorithm (blue) required access to both the output *and* the CNN internals.
+* **The algorithm implementation is tightly coupled to the CNN architecture exposed by the API**. Although the *method* may be general, any particular *implementation* will require the CNN architecture to conform to the specifics of the API.
+
+In general, pros and cons of white box approaches are:
+
+Pros
```

Comment:
Another pro is generally only requires a single forward/backward pass through the model, so it is more computationally efficient compared to black-box explanation methods.